EventGenerator/inc/PositionSamplerTool.hh

```diff
@@ -0,0 +1,43 @@
+// art tool to sample starting positions/times, according to some rule
+// Ed Callaghan, 2024
+
+#ifndef EventGenerator_PositionSamplerTool_hh
+#define EventGenerator_PositionSamplerTool_hh
+
+// stl
+#include <utility>
+#include <vector>
+
+// art
+#include <art/Framework/Principal/Event.h>
```

Comment:
Same comment about < vs " as in the other PR.

Reply:
Done.